sem_covid/services/model_registry.py

```diff
@@ -19,6 +19,7 @@
 SUBCATEGORY_CLASS_EXPERIMENT_ID = '20'
 TYPE_OF_MEASURE_EXPERIMENT_ID = '21'
 TARGET_GROUPS_EXPERIMENT_ID = '22'
+FUNDING_EXPERIMENT_ID = '73'
```

Comment:
:))))

73

Reply:
can we somehow identify experiments by their given name, over which we have control, rather than ID?